src/portal/src/app/base/left-side-nav/clearing-job/audit-log-purge/set-job/set-job.component.html

```
@@ -188,28 +206,37 @@
-                [disabled]="
-                    disableGC ||
-                    purgeForm.invalid ||
-                    !(selectedEventTypes?.length > 0)
-                ">
+                [disabled]="disableGC || !isFormValid()">
                 {{ 'CLEARANCES.PURGE_NOW' | translate }}
             </button>
         </div>
         <div class="clr-col">
+            <button
+                id="gc-now"
```

HTMLHint (1.9.2)

[error] The id value [ gc-now ] must be unique.

(id-unique)